### DISCUSSION TITLE
삭제

### DISCUSSION THREAD

**Discussion by whycallme** [ebandal/H2Orestart#41] [Q&A]

**Comment by ebandal** [comment]
LibreOffice에서  PC에 설치된 JRE 를 인식하지 못하는 문제 때문일것 같습니다.  다른 JRE를 설치해보세요.
동일한 문제로 등록되었던 issue가 있습니다.
[#40](url) 을 확인하세요.

**Reply by whycallme** [reply]
정말 감사합니다. JRE 다시 설치하여 해결 하였습니다.
